shared/chat/conversation/command-markdown/container.tsx

```diff
@@ -6,7 +6,7 @@ type OwnProps = {
   conversationIDKey: Types.ConversationIDKey
 }
 
-export default Container.namedConnect(
+export default Container.connect(
```

Comment:
`namedConnect` is now deprecated. it was only useful for mocking and that layer was entirely broken awhile ago due to react-redux changes. so all it did was give us untyped connect calls